common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
> If you must disable HTML-escaping, of the two alternatives above, using HTML() is preferred, unless the context is ambiguous and HTML() does not make sense, such as in certain Mako defs.

@robrap  Do we consider such cases where the value could be text or html, as ambiguous ?